### DISCUSSION TITLE
Option to Set Playback Framerate Independent of Anti-Flicker Region

### DISCUSSION THREAD

**Discussion by AlexLandherr** [gopro/labs#553] [Feature Requests]
I am wondering whether it would be possible to set the playback fps independently of the "anti-flicker" NTSC/PAL region setting for time lapses when recording in "video" mode?

As an example; I live in Sweden and therefore am using the anti-flicker setting of "50 Hz" and that results in all time lapses using a playback fps of 25.

I would like to be able to select this playback fps before I start a time lapse video recording.

I know I could just record as still images and then animate them with ffmpeg; my request is with those who aren't as proficient and want to combine a greater selection with ease of use and for example get 24 fps playback right of the memory card.

**Comment by dnewman-gpsw** [comment]
Interesting idea.  So feature like oMTFPS=24 (timelapse fps).  For normal Timelapse this should be doable.  However, this wouldn't work well for TimeWarps, which have an option real-time with audio, so those need to be 25/30 (or becomes a much bigger hack.)

**Reply by AlexLandherr** [reply]
When you say normal time lapse does that include Nightlapse, Vehicle trails and Star trails?

I'm thinking that if I have 24 fps = 24 Hz enabled the 24 fps option for this as of yet hypothetical feature would render at exactly 24 fps/Hz.

Also, it should be able to be set permanently like I did for GPS Sync and 24 Hz.
